src/wallet/rpcdump.cpp

```diff
@@ -138,6 +138,11 @@ UniValue importprivkey(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_WALLET_ERROR, "Cannot import private keys to a wallet with private keys disabled");
     }
 
+    LegacyScriptPubKeyMan* spk_man = pwallet->GetLegacyScriptPubKeyMan();
+    if (!spk_man) {
+        throw JSONRPCError(RPC_WALLET_ERROR, "This type of wallet does not support this command");
```

Comment:
nit, could add some helper to do this repeated check (currently 9 times), like `GetLegacyScriptPubKeyMan(pwallet)`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/17260#discussion_r339505236

> nit, could add some helper to do this repeated check (currently 9 times), like `GetLegacyScriptPubKeyMan(pwallet)`?

Would be a nice followup. My preference would be to do it in a later PR though to avoid delay on this one, since this PR is pretty big and already has some acks.

Reply:
Agreed for follow up.